uplink/src/collector/logcat.rs

```diff
@@ -133,14 +134,18 @@ impl LogEntry {
     }
 
     pub fn to_payload(&self, sequence: u32) -> anyhow::Result<Payload> {
-        let payload = serde_json::to_value(self)?;
-
         Ok(Payload {
             stream: "logs".to_string(),
             device_id: None,
             sequence,
             timestamp: self.timestamp,
```

Comment:
This should be log's timestamp if available

Reply:
If logs are expected to have timestamps, can we treat timestamp errors as failures?

Reply:
We only read real time logcat output so the two timestamps will always be very close.

Reply:
Logcat line regex specifies time field as mandatory so if timestamp isn't there the regex match will fail.

Reply:
Flush timeouts are working for logs stream now so the logs issue reactlabs saw shouldn't happen.